scripts/bundle.ts

```diff
@@ -48,6 +48,6 @@ buildSync({
   minify: true,
   splitting: true,
   format: 'esm',
-  target: 'node14',
+  target: 'es2019',
```

Comment:
It's possible to set multiple targets and the lowest one wins. So we should use e.g. this:
```suggestion
  target: ['es2019', 'node14.6'],
```
Sadly I already tried node14.6 and the output still was not compatible with our tests when running on node14.6, but it was green up from node14.13+ 🤔